### ISSUE TITLE
Implement missing DataFrame/Series/Index APIs

### ISSUE THREAD

**Issue by HyukjinKwon** [databricks/koalas#362]
I am opening this as a reminder. Probably I might keep opening and closing for each release - not sure yet about it. I don't intend to open this so long time. 

To get to the point, eventually we should fix all of the missing APIs here:
https://github.com/databricks/koalas/blob/master/databricks/koalas/missing/series.py
https://github.com/databricks/koalas/blob/master/databricks/koalas/missing/frame.py
https://github.com/databricks/koalas/blob/master/databricks/koalas/missing/indexes.py
https://github.com/databricks/koalas/blob/master/databricks/koalas/missing/groupby.py

**Please open a separate issue to start to work on implementing them.**

**Comment by abishekganesh72** [comment]
@HyukjinKwon any particular functions that need to be done first?

**Comment by HyukjinKwon** [comment]
I don't have in my mind yet. We should implement all of them and I wanted to pin the reminder for now :).

**Comment by HyukjinKwon** [comment]
hey @garawalid, let's file a separate issue next time just for separate trackability

**Comment by garawalid** [comment]
@HyukjinKwon Okay. I'll open one that contains missing methods for **10-minute tutorial with Koalas** (#196)

**Comment by shril** [comment]
@HyukjinKwon, @garawalid, @abishekganesh72, for the sake of convenience, let's pick up 10 functions for now and in the meantime, we can add more to this list.
This will ultimately help us with better communication between contributors.

I am selecting 10 frequently used unsupported functions for frame.py - 

- [ ] add
- [ ] aggregate
- [ ] dot
- [ ] eq
- [ ] first
- [ ] keys
- [ ] last
- [ ] info
- [ ] items
- [ ] le

**Comment by shril** [comment]
I am starting with `DataFrame.first()`, `DataFrame.last()` and `DataFrame.keys()`.

**Comment by HyukjinKwon** [comment]
sounds good :-). Actually, can you get rid of `add` in the list? I plan to do them all like https://github.com/databricks/koalas/pull/401

I plan to add those:

- [ ] add
- [ ] radd
- [ ] div
- [ ] divide
- [ ] rdiv
- [ ] truediv
- [ ] rtruediv
- [ ] mul
- [ ] multiply
- [ ] rmul
- [ ] sub
- [ ] substract
- [ ] rsub

soon at frame.py in one go.

If somebody takes this one, then it's even better to me :-).

**Comment by shril** [comment]
@HyukjinKwon, I already have 2 PRs on my plate. 
I will take it once I complete my set of functions.

**Comment by HyukjinKwon** [comment]
I opened a PR for mine, #427

**Comment by manuzhang** [comment]
@shril Any progress on your remaining functions ?

**Comment by HyukjinKwon** [comment]
@manuzhang feel free to take over if we don't get response within few days.

**Comment by xinrong-databricks** [comment]
Hi @HyukjinKwon, since we have https://github.com/databricks/koalas/issues/1929 for tracking missing API, shall we close this issue? Thank!

**Comment by itholic** [comment]
> Hi @HyukjinKwon, since we have #1929 for tracking missing API, shall we close this issue? Thank!

+1
